packages/yoga/src/Avatar/web/Avatar.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { BuildingFilled } from '@gympass/yoga-icons';
+import { string, func, checkPropTypes } from 'prop-types';
+
+import Box from '../../Box';
+import Icon from '../../Icon';
+
+const Avatar = ({ src, alt, fill, stroke, placeholder, ...otherProps }) => (
+  <Box
+    bgColor="elements.selectionAndIcons"
+    display="flex"
+    alignItems="center"
+    justifyContent="center"
+    width={48}
+    height={48}
+    borderRadius="small"
+    elevation="small"
+    overflow="hidden"
+    {...otherProps}
+  >
+    {src ? (
+      <img src={src} alt={alt} />
+    ) : (
+      <Icon
+        as={placeholder}
```

Comment:
What do we think about calling this `placeholder`? When I read it I thought about the Input placeholder attribute.

What do you think about naming it `icon`?

**Before:**
```jsx
import { YouTube } from '@gympass/yoga-icons';

<Avatar placeholder={YouTube} />
```

**After:**
```jsx
import { YouTube } from '@gympass/yoga-icons';

<Avatar icon={YouTube} />
```